dev/managedservicesplatform/operationdocs/operationdocs.go

```diff
@@ -83,29 +83,47 @@ This service is operated on the %s.`,
 	serviceKind := pointers.Deref(s.Service.Kind, spec.ServiceKindService)
 	serviceDirURL := fmt.Sprintf("https://github.com/sourcegraph/managed-services/blob/main/services/%s", s.Service.ID)
 	serviceConfigURL := fmt.Sprintf("%s/service.yaml", serviceDirURL)
+
+	serviceDetails := [][]string{
+		{"Service ID", fmt.Sprintf("%s (%s)",
+			markdown.Code(s.Service.ID), markdown.Link("specification", serviceConfigURL))},
+		// TODO: See service.Description docstring
+		// {"Description", s.Service.Description},
+		{"Owners", strings.Join(mapTo(s.Service.Owners, markdown.Bold), ", ")},
+		{"Service kind", fmt.Sprintf("Cloud Run %s", string(serviceKind))},
+		{"Environments", strings.Join(mapTo(s.Environments, func(e spec.EnvironmentSpec) string {
+			l, h := markdown.HeadingLinkf(e.ID)
+			environmentHeaders = append(environmentHeaders, environmentHeader{
+				environmentID: e.ID,
+				header:        h,
+				link:          l,
+			})
+			return l
+		}), ", ")},
+		{"Docker image", markdown.Code(s.Build.Image)},
+		{"Source code", markdown.Linkf(
+			fmt.Sprintf("%s - %s", markdown.Code(s.Build.Source.Repo), markdown.Code(s.Build.Source.Dir)),
+			"https://%s/tree/HEAD/%s", s.Build.Source.Repo, path.Clean(s.Build.Source.Dir))},
+	}
+
+	region := "us-central1"
+	if s.Rollout != nil {
+		finalStageEnv := s.Rollout.Stages[len(s.Rollout.Stages)-1].EnvironmentID
+		var finalStageProject string
+		for _, environment := range s.Environments {
+			if environment.ID == finalStageEnv {
+				finalStageProject = environment.ProjectID
+				break
+			}
+		}
+		serviceDetails = append(serviceDetails,
+			[]string{"Rollout Pipeline", markdown.Linkf(fmt.Sprintf("`%s-%s-rollout`", s.Service.ID, region),
```

Comment:
To align with the others, and I think we generally use sentence casing for titles:

```suggestion
			[]string{"Rollout pipeline", markdown.Linkf(fmt.Sprintf("`%s-%s-rollout`", s.Service.ID, region),
```